metricbeat/module/traefik/health/_meta/fields.yml

```diff
@@ -0,0 +1,45 @@
+- name: health
+  type: group
+  description: >
+    Metrics obtained from Traefik's health API endpoint
+  fields:
+    - name: uptime
+      type: group
+      description: >
+        Uptime of Traefik instance
+      fields:
+        - name: sec
+          type: int
+          description: >
+            Uptime in seconds
+    - name: response
+      type: group
+      description: >
+        Response metrics
+      fields:
+        - name: count
+          type: int
+          description: >
+            Number of responses
+        - name: avg_time
+          type: group
+          description: >
+            Average response time
+          fields:
+            - name: sec
+              type: float
+              description:
+                  Average response time in seconds
+        - name: status_codes
+          type: group
+          description: >
+            Respone status codes
+          fields:
+            - name: code
```

Comment:
Not sure if this is correct. If understand the code correct it will be `status_codes.404: 17`? So `code` is actually the key and not the value? For such cases have a look at the dynamic templates. Search for `object_type` usage in our fields.yml to see how it can be used.